src/util/options.schema.json

```diff
@@ -31,36 +31,42 @@
     },
     "verify": {
       "title": "verify",
-      "description":
-        "Verify that the solver reached a post-fixpoint. Beware that disabling this also disables output of warnings since post-processing of the results is done in the verification phase!",
+      "description": "Verify that the solver reached a post-fixpoint. Beware that disabling this also disables output of warnings since post-processing of the results is done in the verification phase!",
```

Comment:
Apparently d8b94a5c5f936d9d3a0733003de84b42fa728d32 reformatted all the schema...
Maybe I'll rebase something here.

Reply:
Done! Now the only schema change is the single added option.
Interactive rebase with merges is not for the faint-hearted.